templates/docs/patterns/linked-logo-section/index.md

```diff
@@ -0,0 +1,210 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Linked logo section | Patterns
+---
+
+{% from "docs/macros/patterns/wip-notice.jinja" import pattern_wip_notice %}
+
+{{- pattern_wip_notice() }}
+
+A linked logo section is used to display a list of logos which explicitly link to additional pages or resources.
+
+The linked logo section pattern is composed of the following elements:
+
+| Element                           | Description                                                                               |
+| --------------------------------- | ----------------------------------------------------------------------------------------- |
+| title_text (**required**)         | `H2` title text.                                                                          |
+| Layout                            | Defaults to `full-width`, with additional options for `50-50`, and `25-75` layout splits. |
+| Links (**required**)              | An `Array<Object>` of individual image link properties.                                   |
+| Links[].href (**required**)       | The target link for the logo.                                                             |
+| Links[].text (**required**)       | The link text for the logo.                                                               |
+| Links[].label (**required**)      | The `aria-label` for the logo.                                                            |
+| Links[].image_html (**required**) | The logo image element.                                                                   |
+
+## Full width
```

Comment:
In the 50/50 and 25/75 sections, you noted the max number of logos per variant, but you haven't noted that here. It's worth nothing here that the full width variant can accept  8 logos.